new/client-rb-logger.md

```diff
@@ -0,0 +1,39 @@
+---
+RFC: unassigned
+Title: Logging for chient.rb
+Author: Zeal Jagannatha <zeal@fb.com>
+Status: Draft
+Type: Standards Track
+Tracking:
+  - https://github.com/chef/chef/issues/5526
+---
+
+# Logging for client.rb
```

Comment:
solo.rb yes, since it's just a client.rb with a few special options.

I'm not sure about knife config files.